Packs/FeedTAXII/ReleaseNotes/1_2_10.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+Changed the condition used to determine, which version of TAXII 2 to use - 2.0 or 2.1.
```

Comment:
```suggestion
- Updated to latest version of ApiModules.
```

Reply:
Here the same as above, "-" should be removed.